examples/front_ends/simple_auth/src/simple_auth/configs/config.yml

```diff
@@ -40,6 +46,34 @@ llms:
     max_tokens: 1024
 
 authentication:
+  jira:
+    _type: oauth2_auth_code_flow
+    client_url: "http://localhost:8000"
+    authorization_url: https://auth.atlassian.com/authorize
+    token_url: https://auth.atlassian.com/oauth/token
+    client_id: ${AIQ_OAUTH_CLIENT_ID}
+    client_secret: ${AIQ_OAUTH_CLIENT_SECRET}
+    token_endpoint_auth_method: client_secret_post
+    response_type: code
+    authorization_kwargs:
+      prompt: consent
+      audience: api.atlassian.com
+    scopes:
+      - report:personal-data
+      - read:jira-work
+      - manage:jira-project
+      - manage:jira-configuration
+      - read:jira-user
+      - write:jira-work
+      - manage:jira-webhook
+      - manage:jira-data-provider
+      - read:servicedesk-request
+      - manage:servicedesk-customer
+      - write:servicedesk-request
+      - read:servicemanagement-insight-objects
+      - "offline_access"
+    use_pkce: false
+
```

Comment:
This will prevent the example from running on its own without a Jira instance the user can test. I would prefer a second config or including it in a different example.